deps/spidershim/tests.gyp

```diff
@@ -28,5 +28,9 @@
       'target_name': 'value',
       'sources': [ 'test/value.cc' ],
     },
+    {
+      'target_name': 'v8v8',
```

Comment:
Nit: For consistency, please rename this to `v8` and `v8.cc`.  We don't use the "v8" prefix for test file names.